libs/forecasting/src/lib/ForecastingDashboard/Controls/ForecastComparison.tsx

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Stack, Text } from "@fluentui/react";
+import {
+  ModelAssessmentContext,
+  defaultModelAssessmentContext,
+  BasicHighChart,
+  JointDataset,
+  orderByTime
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import { SeriesOptionsType } from "highcharts";
+import React from "react";
+
+import { forecastingDashboardStyles } from "../ForecastingDashboard.styles";
+
+import { getForecastPrediction } from "./getForecastPrediction";
+
+export class IForecastComparisonProps {}
+
+export interface IForecastComparisonState {
+  timeSeriesId?: number;
+  baselinePrediction?: number[];
+}
+
+const stackTokens = {
```

Comment:
nit: should we move this to forecastingDashboardStyles?

Reply:
AFAIK stack tokens don't quite work the same way, but I would love to be corrected on this as it really should live in a style file. If you have an example where we do it that way I will be happy to make the PR conform 🙂

Reply:
I checked a dozen examples and this seems to be the way we're doing it everywhere else for stack tokens. I'm marking this one as resolved, but do feel free to reopen if you have a suggestion for a cleaner way!